rsmtool/preprocessor.py

```diff
@@ -1132,6 +1142,10 @@ def preprocess_features(
             Whether we should use the truncation set
             for removing outliers.
             Defaults to ``False``.
+        truncate_outliers : bool, optional
```

Comment:
maybe pass the flag before `use_trancations`? It will make more sense when reading the docstring.